slices/python3-minimal.yaml

```diff
@@ -0,0 +1,29 @@
+package: python3-minimal
+
+essential:
+  - python3-minimal_copyright
+
+slices:
+  bins:
+    essential:
+      - python3-minimal_config
+      - python3-minimal_modules
+      - python3.12-minimal_bins
+    contents:
+      /usr/bin/py3clean:
+      /usr/bin/py3compile:
+      /usr/bin/py3versions:
```

Comment:
do we need these? 

If we want to keep the `bins` slice, all of these must be tested.

In the end, we only care about the python3 symlink right? If so, I'd create a dedicated slice for it (and test it).

Reply:
Added more slices.

Reply:
Thanks.

We need to be consistent about `bins`.

As per other slices, `bins` typically deliver all binaries.

In this case, we want the `/usr/bin/python3` symlink, so we should have a dedicated slice for it.

You've created dedicated slices for the other bins too, which is ok.